playground/backend/cmd/server/controller_test.go

```diff
@@ -863,7 +863,7 @@ func TestPlaygroundController_SaveSnippet(t *testing.T) {
 				t.Errorf("PlaygroundController_SaveSnippet() error = %v, wantErr %v", err, tt.wantErr)
 			}
 			if err == nil {
-				if len(got.Id) != 11 || got.Id != tt.wantId {
+				if len(got.Id) != 11 {
```

Comment:
why is the length always 11?

Reply:
It's the length of the identifier that is used to store snippets in the Cloud Datastore. It's appropriate length to save storage size in the Cloud Datastore and provide good randomnicity. This parameter stores in the playground/backend/properties.yaml file